setup.py

```diff
@@ -38,16 +38,41 @@
         # Indicate who your project is intended for
         'Intended Audience :: Science/Research',
         'Intended Audience :: Education',
+        'Topic :: Scientific/Engineering :: Atmospheric Science',
 
         # Pick your license as you wish (should match 'license' above)
-        'License :: OSI Approved :: BSD License',
+        'License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)'
 
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.,
-        'Programming Language :: Python :: 3.6'
+        'Programming Language :: Python :: 3 :: Only',
+        'Programming Language :: Python :: 3.6',
```

Comment:
maybe we should drop support for 3.6?

Reply:
OK, but not on this PR.
I can write a new one once is PR is merged.

Reply:
Unless we need to drop 3.6 to pass CI

Reply:
it seems dumping python 3.6 wont be nessesary